modules/search/instant-search/components/product-ratings.jsx

```diff
@@ -0,0 +1,36 @@
+/** @jsx h */
+/**
+ * External dependencies
+ */
+import { h } from 'preact';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from './gridicon';
+
+/**
+ * Style dependencies
+ */
+import './product-ratings.scss';
+
+/**
+ * Renders a hook-based component for displaying product ratings.
+ *
+ * @param {object} props - Component properties.
+ * @param {number} props.count - Number of ratings.
+ * @param {number} props.rating - Average rating out of five.
+ * @returns {object} Product rating component.
+ */
+export default function ProductRatings( { rating = 0, count = 0 } ) {
+	return (
+		<div className="jetpack-instant-search__search-result-product-rating">
+			<span className="jetpack-instant-search__search-result-product-rating-stars">
```

Comment:
I think we need to consider how this would work in a screenreader. How about `aria-hidden` on the stars themselves, then adding some descriptive text after the count hidden with the class `screen-reader-text`?

Reply:
👍 Went ahead and implemented something similar.